pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/Properties/Resources.resx

```diff
@@ -784,6 +784,9 @@
   <data name="MainForm_TryExecuteOperation_Cannot_add___0___because_there_is_another_configuration_with_the_same_name_" xml:space="preserve">
     <value>Cannot add "{0}" because there is another configuration with the same name.</value>
   </data>
+  <data name="MainForm_btnOpenTemplate_Click_The_template_file_for___0___has_not_been_downloaded__Please_run___0___and_try_again_" xml:space="preserve">
+    <value>The template file for "{0}" has not been downloaded. Please run "{0}" and try again.</value>

```

Comment:
Is this an unused string? I can't find any code that uses MainForm_btnOpenTemplate_Click_The_template_file_for___0___has_not_been_downloaded__Please_run___0___and_try_again_